Docs: Fixes and refinements in the integration guides.

gorzelinski opened the pull request:
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Fixes and refinements in the integration guides.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

gorzelinski commented:
I tried to find typos, mistakes, and inconsistencies in the integration guides. I fixed them (I think), but more pairs of eyes would be helpful.

filipsobol commented on docs/getting-started/integrations-cdn/vuejs-v3.md:
```diff
@@ -305,11 +307,11 @@ Since accessing the editor toolbar is not possible until after the editor instan
-			v-model="data"
-			:editor="editor"
-			@ready="onReady"
-		/>
+		v-if="editor"
+		v-model="data"
+		:editor="editor"
+		@ready="onReady"
+	/>
```
The old indention is correct

gorzelinski replied:
It looks strange, tbh. But you're the boss in Vue :D

filipsobol replied:
Indention is off again :smile:

gorzelinski replied:
Earlier, in this guide, it looks like this:

```html
<template>
	<ckeditor
		v-if="editor"
		v-model="data"
		:editor="editor"
		:config="config"
	/>
</template>
```

Isn't it unified now? Am I stupid? :D

filipsobol replied:
It displays like this on GitHub

```html
    <ckeditor
		v-if="editor"
		v-model="data"
		:editor="editor"
		@ready="onReady"
	/>
```

Maybe it's caused by mixing spaces and tabs?

gorzelinski replied:
Ok, I have my editor configured for tabs. But for some reason, there were some spaces. The snippet was probably off after multiple people copied and pasted it. Now, it should be ok.

godai78 commented on docs/getting-started/integrations-cdn/angular.md:
```diff
@@ -18,6 +18,8 @@ order: 30
 
 Angular is a TypeScript-based, open-source, single-page web application framework. The CKEditor 5 component for Angular supports integrating different editor types.
 
+This guide assumes you already have a Angular project. To create such a project, you can use Angular CLI. Refer to the [Angular documentation](https://angular.io/cli) to learn more.
```
```suggestion
This guide assumes you already have an Angular project. To create such a project, you can use Angular CLI. Refer to the [Angular documentation](https://angular.io/cli) to learn more.
```

gorzelinski replied:
I don't know if it's a good place for this paragraph, but I would keep it in some form. Some people were confused when we didn't explicitly say: "Hey, you need a project first."

godai78 replied:
I would keep it, too. I believe the place is as good as any before describing the procedure.

pszczesniak reviewed (commented):
In all code snippets where there are loaded `Essentials` and `Undo` plugins - `Undo` is already in `Essentials`.

https://ckeditor.com/docs/ckeditor5/latest/api/module_essentials_essentials-Essentials.html

gorzelinski commented on docs/getting-started/integrations-cdn/react-default-cdn.md:
```diff
@@ -177,59 +178,14 @@ function CKEditorNestedInstanceDemo( { name, content } ) {
-					CKEDITOR.TextTransformation,
-					CKEDITOR.Base64UploadAdapter
+					CKEDITOR.Undo,
+					CKEDITOR.Mention
```
I'll change it 👍

gorzelinski commented:
> In all code snippets where there are loaded `Essentials` and `Undo` plugins - `Undo` is already in `Essentials`.
> 
> https://ckeditor.com/docs/ckeditor5/latest/api/module_essentials_essentials-Essentials.html

I didn't know. So, now it's redundant. I'll remove them.

pszczesniak commented on docs/getting-started/integrations-cdn/angular.md:
```diff
@@ -716,27 +655,25 @@ export class AppComponent {
-			toolbar: {
-				items: [ 'undo', 'redo', '|', 'bold', 'italic' ],
-			}
+			plugins: [ Essentials, Bold, Italic, Paragraph, Undo ],
```
```suggestion
			plugins: [ Essentials, Bold, Italic, Paragraph ],
```

filipsobol commented on docs/getting-started/integrations-cdn/vuejs-v3.md:
```diff
@@ -310,6 +310,7 @@ Since accessing the editor toolbar is not possible until after the editor instan
 			:editor="editor"
 			@ready="onReady"
 		/>
+	</ckeditor>
```
The tag above is self-closing (ending with `/>`), so the closing tag is not needed (even incorrect).

gorzelinski commented:
I've removed the Vue 2 CDN guide. I rebuilt the docs, and everything works fine. But there is still a guide for Vue 2 with npm. I updated it, and it works with NIM, but I wonder if we want asymmetry. WDYT?

godai78 commented:
> I've removed the Vue 2 CDN guide. I rebuilt the docs, and everything works fine. But there is still a guide for Vue 2 with npm. I updated it, and it works with NIM, but I wonder if we want asymmetry. WDYT?

Let's keep it if it works.

gorzelinski commented:
> Let's keep it if it works.

Yup, I updated it earlier.

filipsobol commented:
> > Let's keep it if it works.
> 
> Yup, I updated it earlier.

I don't think it works with NIM because it wasn't migrated to ESM like Vue 3 integration. I think we should remove it from both npm and cloud and only keep it in the “Legacy” section.

gorzelinski commented:
> I don't think it works with NIM because it wasn't migrated to ESM like Vue 3 integration. I think we should remove it from both npm and cloud and only keep it in the “Legacy” section.

Ok, that's even better. Will do.

gorzelinski commented on docs/getting-started/index.md:
```diff
@@ -45,7 +45,7 @@ Do you want to use a framework? Native integrations with the most popular librar
 * Integrate CKEditor&nbsp;5 with Next.js using {@link getting-started/integrations-cdn/next-js CDN} or {@link getting-started/integrations/next-js npm/ZIP}
 * Integrate CKEditor&nbsp;5 with Vue.js 3.x using {@link getting-started/integrations-cdn/vuejs-v3 CDN} or {@link getting-started/integrations/vuejs-v3 npm/ZIP}
-* Integrate CKEditor&nbsp;5 with Vue.js 2.x using {@link getting-started/integrations-cdn/vuejs-v2 CDN} or {@link getting-started/integrations/vuejs-v2 npm/ZIP}
+* Integrate CKEditor&nbsp;5 with Vue.js 2.x using {@link getting-started/integrations/vuejs-v2 npm/ZIP}
```
Yeah, I found like 2 links.

godai78 replied:
Yeah, I removed the CDN one, but we kept the npm guide as it actually works. Did we remove the code package from npm, too?
